apps/vr-tests-react-components/src/stories/Radio.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30207&commitId=775057c67ee763f43b79fbcc824178c86a1d2103&env=prod&clientType=FLUENTUI)


  <details><summary><b>Radio Converged</b> 23 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio Converged.checked - Dark Mode.active.chromium.png|2| Changed|
|Radio Converged.checked - Dark Mode.hover.chromium.png|3| Changed|
|Radio Converged.checked.hover.chromium.png|4| Changed|
|Radio Converged.disabled+checked.default.chromium.png|2| Changed|
|Radio Converged.group-horizontal-stacked - RTL.default.chromium.png|4| Changed|
|Radio Converged.checked - Dark Mode.rest.chromium.png|2| Changed|
|Radio Converged.checked.rest.chromium.png|4| Changed|
|Radio Converged.group-horizontal.default.chromium.png|4| Changed|
|Radio Converged.group-vertical - RTL.default.chromium.png|4| Changed|
|Radio Converged.checked - High Contrast.hover.chromium.png|4| Changed|
|Radio Converged.checked.active.chromium.png|5| Changed|
|Radio Converged.checked - High Contrast.active.chromium.png|4| Changed|
|Radio Converged.group-horizontal - RTL.default.chromium.png|4| Changed|
|Radio Converged.checked - High Contrast.rest.chromium.png|4| Changed|
|Radio Converged.disabled+checked - High Contrast.default.chromium.png|4| Changed|
|Radio Converged.group-horizontal-stacked.default.chromium.png|4| Changed|
|Radio Converged.group-vertical.default.chromium.png|4| Changed|
|Radio Converged.group-disabled - RTL.default.chromium.png|2| Changed|
|Radio Converged.group-disabled.default.chromium.png|2| Changed|
|Radio Converged.indicator+checked.active.chromium.png|0| Added|
|Radio Converged.indicator+checked.hover.chromium.png|0| Added|
|Radio Converged.indicator+checked.rest.chromium.png|0| Added|
|Radio Converged.indicator+unchecked.default.chromium.png|0| Added|

</details>


  <details><summary><b>Table flex - subtle selection</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|4| Changed|
|Table flex - subtle selection.rest.chromium.png|4| Changed|

</details>


  <details><summary><b>Table layout flex</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|3| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|4| Changed|
|Table layout flex.single select (checked).chromium.png|4| Changed|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|4| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|3| Changed|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|2| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|4| Changed|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|2| Changed|
|Table layout flex.single select (checked) - brand.chromium.png|4| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|3| Changed|
|Table layout flex.single select (checked) - RTL.chromium.png|4| Changed|

</details>


  <details><summary><b>Table layout table</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table.single select (checked) - RTL.chromium.png|4| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|4| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|4| Changed|
|Table layout table.single select (checked) - brand.chromium.png|4| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|4| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|3| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|2| Changed|
|Table layout table.single select (checked).chromium.png|4| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|3| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|3| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|2| Changed|

</details>


  <details><summary><b>Table table - subtle selection</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table table - subtle selection.rest.chromium.png|4| Changed|
|Table table - subtle selection.rest.hover unselected row.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
These look good to me but a second set of eyes is probably a good thing here.

Reply:
Changes LGTM as well. The new indicator circle is a fraction of a pixel larger, but its radius is still ~10px, which is what is in the figma.